gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_3_0/schema.yml

```diff
@@ -0,0 +1,15 @@
+databaseChangeLog:
+  - changeSet:
+      id: 4.3.0
+      author: GraviteeSource Team
+      changes:
+        # ################
+        # OrganizationLicense changes
+        # ################
+        - createTable:
+            tableName: ${gravitee_prefix}organization_licenses
+            columns:
+              - column: {name: organization_id, type: nvarchar(64), constraints: { nullable: false, primaryKey: true, primaryKeyName: pk_organization_license } }
+              - column: {name: license, type: nvarchar(64), constraints: { nullable: false } }
```

Comment:
Are we sure 64 characters is enough for a license?

Reply:
I'll bump it to 256. I have a license locally that is 616 bytes, which would be 206 characters in base64.